app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/recordaudio/RecordAudioButtons.kt

```diff
@@ -100,6 +104,14 @@ fun RecordAudioButtonRecording(
             seconds += 1
         }
     }
+    val activity = LocalContext.current as Activity
```

Comment:
There is a `LocalActivity` already, so it can be even `val activity = LocalActivity.current` 😎